cql2pgjson/src/test/java/org/folio/cql2pgjson/ForeignKeyGenerationTest.java

```diff
@@ -123,4 +126,24 @@ public void invalidCurrentTable() throws Exception {
     assertTrue(sql, sql.contains("'tableb'"));
   }
 
+  @Test
+  public void testSearchInstanceByItemBarcode() throws Exception {
+    CQL2PgJSON cql2pgJson = new CQL2PgJSON("instance");
+    cql2pgJson.setDbSchemaPath("templates/db_scripts/subquery.json");
+    String sql = cql2pgJson.toSql("item.barcode == 7834324634").toString();
+    String expected = "WHERE instance.id IN  ( SELECT (holdings_record.jsonb->>'instanceId')::UUID from holdings_record WHERE holdings_record.id IN  ( SELECT (item.jsonb->>'holdingsRecordId')::UUID from item WHERE lower(f_unaccent(item.jsonb->>'barcode')) LIKE lower(f_unaccent('7834324634'))))";
+    assertEquals(expected, sql);
+    System.out.println(sql);
```

Comment:
Can you remove the println? This is redundant because it either matched expected or is reported by assertEquals.

Reply:
done